src/client/sound/ogg.c

```diff
@@ -323,6 +324,24 @@ OGG_Stream(void)
 			}
 		}
 	}
+
+	if (ogg_status == PLAY && ogg_shuffle->modified)
+	{
+		// Shuffle was modified before last track finished.
+		ogg_shuffle->modified = false;
+	}
+	else if (ogg_status == STOP && ogg_shuffle->modified)
+	{
+		// If fullscreen console, clear the map cdtrack.
+		if (cls.state == ca_disconnected)
+		{
+			ogg_mapcdtrack = 0;			// Track 0 means "stop music".
```

Comment:
Does this work as intended with `ogg_ignoretrack0 1`?

Reply:
Yes. That statement is to clear the cd track if the user had played a map and disconnected, then changed any of the playback parameters through the console\menu after playback of the last track had finished.

Reply:
Ah, okay.